src/pages/yeonokkang/Login/Input.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default class Input extends React.Component {
+  render() {
+    const { activated, handleInput, value, type, placeholder } = this.props;
+    return (
+      <>
+        <input
```

Comment:
```suggestion
    return (
      <input
```
Fragment는 가장 바깥쪽 요소가 두개 이상일 때만 사용하는 요소입니다. 이 경우에는 input 태그 하나만 존재하기 때문에 불필요해보입니다.

Reply:
확인 후 수정 완료했습니다  😊